packages/suite/src/views/wallet/transactions/TransactionList/WalletTransactionList.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+
+import styled from 'styled-components';
+
+import { Card, Column } from '@trezor/components';
+import { typography } from '@trezor/theme';
+
+import { Translation } from 'src/components/suite';
+import { Account, WalletAccountTransaction } from 'src/types/wallet';
+
+import { TransactionList } from './TransactionList';
+import { useVisibleTransactionsRetriever } from './useFetchTransactions';
+
+const NoResults = styled.div`
```

Comment:
@jvaclavik pls advice, taht's the clsoes want and most similar I found, but it obviously violates the design system doctrine.

Reply:
See below